src/locales/en/messages.po

```diff
@@ -494,10 +494,6 @@ msgstr "Cancel"
 msgid "Cannot disable E-Mode"
 msgstr "Cannot disable E-Mode"
 
-#: src/components/transactions/Supply/SupplyModalContent.tsx
-msgid "Cap reached. Lower supply amount"
```

Comment:
This is removed along with the `blockingError` logic in `SupplyModalContent` because this state should be unreachable when max amount factors in the available supply cap